markets/perps-market/contracts/storage/PerpsAccount.sol

```diff
@@ -211,9 +217,9 @@ library PerpsAccount {
     /**
      * @notice  This function returns the minimum margin an account requires to stay above liquidation threshold
      */
-    function getAccountMaintenanceMargin(
+    function getAccountRequiredMargins(
         Data storage self
-    ) internal view returns (uint accountMaintenanceMargin) {
+    ) internal view returns (uint accountMaintenanceMargin, uint initialMaintenanceMargin) {
```

Comment:
The naming looks backwards here compared to the return, and compared to where it is called on `getAccountRequiredMargins`